src/components/dashboard/SendQRSummary.js

```diff
@@ -70,8 +71,9 @@ const SendQRSummary = (props: AmountProps) => {
         <Section.Row style={styles.sectionRow}>
           <Section.Title style={styles.headline}>Summary</Section.Title>
           <View style={styles.sectionTo}>
-            <Avatar size={90} />
+            <Avatar.Image size={90} style={{ backgroundColor: 'white' }} source={profile && profile.avatar} />
```

Comment:
@kbardi shouldn't we use UserAvatar?

Reply:
I cannot use UserAvatar because it also renders fullName, but I'm using now Avatar component from common's folder

Reply:
@kbardi @dminones should we extend UserProfile with render options? ie not to render full name? decide on best approach